src/subscriptions/SubscriptionsScreen.js

```diff
@@ -17,6 +17,17 @@ class SubscriptionsScreen extends React.Component {
     subscriptions: [],
   };
 
+  state: {
+    filter: string,
+  };
+
+  constructor() {
+    super();
+    this.state = {
+      filter: '',
```

Comment:
I'm slightly confused as to what's happening here - why do we use a `state: { filter: string,};` and then use `this.state` in `constructor`? Is it some kind of React magic I should learn? :)